docs/glossary.rst

```diff
@@ -12,10 +12,10 @@ Astropy Glossary
       parenthesized ``n``.
       This is from NumPy; see https://numpy.org/doc/stable/glossary.html#term-n.
 
-   <Class>-like
-      An instance of the ``Class`` or a valid initializer argument for ``Class`` as
-      ``Class(value)``. E.g. :class:`~astropy.units.Quantity`-like includes ``"2 *
-      u.km"`` because ``astropy.units.Quantity("2 * u.km")`` works.
+   -like
```

Comment:
It looks like this line was updated by accident ?

Reply:
The diffs at this level are misleading because they got alphabetized. Best to look at the final version.

Reply:
ah yes, sorry I got mixed up while reviewing something else at the same time !